coralproject/talk
# Revert virtualized rendering + story tree generation

## What does this PR do?

Revert the virtualized rendering and story tree generation for the time being until we can find a more performant solution.

All of the virtualized rendering and story tree code now resides in the `virtualized-rendering` branch.

## What changes to the GraphQL/Database Schema does this PR introduce?

Revert all story tree generation and virtualized rendering specific schema changes introduced via 7.1.0, 7.1.1, and 7.1.2 releases.

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indexes.

## How do I test this PR?

- Check codebase has all virtualized rendering (virtuoso) and story tree generation code reverted
- Start up Coral
- Test admin, see that it works
- Test stream, see that it works
 
## How do we deploy this PR?

- No special considerations
- Should be first priority to merge into develop so that we can keep merging other fixes not related to virtualized rendering or story tree's
